src/backend/distributed/utils/colocation_utils.c

```diff
@@ -96,6 +98,74 @@ mark_tables_colocated(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * update_distributed_table_colocation updates the colocation of a table.
+ * if colocate_with -> 'none' then the table is assigned a new
+ * colocation group.
+ */
+Datum
+update_distributed_table_colocation(PG_FUNCTION_ARGS)
+{
+	Oid destinationRelationId = PG_GETARG_OID(0);
```

Comment:
destination is a bit confusing, since there is a source/current colocation ID and a destination/new colocation ID, but it's the opposite.

Reply:
maybe this should be targetRelationId because `MarkTablesColocated(source, target)` puts target into the colocation group of source.

So when we call `update_distributed_table_colocation('A','B') ` A will be put into the colocation group of B. So A is the target and B is the source. That is what I understand, does that sound good to you?

Reply:
I think target is ok. Rather than source, maybe something more explicit like "colocateWithTableId"